stdlib/jdbc/src/main/ballerina/src/java.jdbc/Module.md

```diff
@@ -337,4 +337,5 @@ if (retCall is ()|table<record {}>[]) {
     error err = retCall;
     io:println("Stored procedure call failed: ", <string>err.detail()["message"]);
 }
-```
\ No newline at end of file
+```
+>**Note:** The default thread pool size used in Ballerina is number of processers available * 2. You can configure the thread pool size by using the `BALLERINA_MAX_POOL_SIZE` environment variable.
```

Comment:
I think it's better the "processers" here should be spelled as "[processors](https://www.intel.com/content/www/us/en/products/processors.html)".